nimble/host/include/host/ble_gap.h

```diff
@@ -973,6 +975,21 @@ struct ble_gap_event {
             uint8_t adv_clk_accuracy;
         } periodic_transfer;
 #endif
+	struct {
```

Comment:
please comments those events and its members as we have for other events

Reply:
you also need to comment what this event is about and to which event type it match,  see other events (it is for user to know which member in union should it use for type)